example/lib/presentation/samples/bar/bar_chart_sample1.dart

```diff
@@ -162,7 +162,35 @@ class BarChartSample1State extends State<BarChartSample1> {
     return BarChartData(
       barTouchData: BarTouchData(
         touchTooltipData: BarTouchTooltipData(
-          tooltipBgColor: Colors.blueGrey,
+          getTooltipColor: (group) {
+            Color bgColor;
+            switch (group.x) {
```

Comment:
Please use [the dart3 switch expression](https://dart.dev/language/branches#switch-expressions) here.
It can be like this:
```dart
Color bgColor = switch(group.x) {
   0 => AppColors.contentColorPurple.withAlpha(100),
   1 => AppColors.contentColorYellow.withAlpha(100),
...
}

Reply:
Done